webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_view_settings.py

```diff
@@ -0,0 +1,196 @@
+            {"label": "No label", "value": "hide"},
+            {
+                "label": "Simple label",
+                "value": "simple",
+            },
+            {
+                "label": "Detailed label",
+                "value": "detailed",
+            },
+        ]
+        self.initial_reference = (
+            reference if reference in self.sensnames else self.sensnames[0]
+        )
+        self.allow_click = allow_click
+
+    def layout(self) -> List[Component]:
+        return [
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.REFERENCE),
+                label="Reference",
+                options=[{"label": r, "value": r} for r in self.sensnames],
+                value=self.initial_reference,
+                multi=False,
+                clearable=False,
+            ),
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.SCALE),
+                label="Scale",
+                options=[{"label": r, "value": r} for r in self.scales],
+                value=self.scales[0],
+                multi=False,
+                clearable=False,
+            ),
+            wcc.SelectWithLabel(
+                id=self.register_component_unique_id(ViewSettings.IDs.SENSITIVITEIS),
+                label="Select sensitivities",
+                options=[{"label": r, "value": r} for r in self.sensnames],
+                value=self.sensnames[0],
+                multi=True,
+            ),
+            html.Button(
+                id=self.register_component_unique_id(ViewSettings.IDs.RESET_BUTTON),
+                title="Reset Selected Sesitivities",
+                style={
+                    "fontSize": "10px",
+                    "marginTop": "10px",
+                }
+                if self.allow_click
+                else {"display": "none"},
+                children="Clear selected",
+            ),
+            wcc.Checklist(
+                id=self.register_component_unique_id(ViewSettings.IDs.PLOT_OPTIONS),
+                label="Plot options",
+                options=self.plot_options,
+                value=[],
+                labelStyle={"display": "block"},
+            ),
+            wcc.Dropdown(
+                id=self.register_component_unique_id(ViewSettings.IDs.LABEL),
+                label="Label",
+                options=self.label_options,
+                value="detailed",
+                clearable=False,
+            ),
+        ]
+
+    def set_callbacks(self) -> None:
+        @callback(
+            Output(
+                self.get_store_unique_id(PlugInIDs.Stores.ViewSetttings.REFERENCE),
+                "data",
+            ),
+            Input(
+                self.component_unique_id(ViewSettings.IDs.REFERENCE).to_string(),
+                "value",
+            ),
+        )
+        def _set_reference(ref: str) -> str:
+            return ref
+
+        @callback(
```

Comment:
This roundtrip is not necessary and should be removed.
I.e. use the component directly in the relevant callback.